mllib/src/main/scala/org/apache/spark/mllib/evaluation/BinaryClassificationMetrics.scala

```diff
@@ -45,6 +45,8 @@ class BinaryClassificationMetrics @Since("3.0.0") (
     @Since("1.3.0") val scoreAndLabels: RDD[_ <: Product],
     @Since("1.3.0") val numBins: Int = 1000)
   extends Logging {
+
+  @deprecated("this variable should be private and will be removed in 4.0.0.", "3.4.0")
```

Comment:
Can we be more specific and say something like "The variable `scoreLabelsWeight` should be private and will be removed in 4.0.0."?

Reply:
sure！